README.md

```diff
@@ -1,5 +1,5 @@
 # Screwdriver Datastore Dynamodb
-[![Version][npm-image]][npm-url] ![Downloads][downloads-image] [![Build Status][wercker-image]][wercker-url] [![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
+[![Version][npm-image]][npm-url] ![Downloads][downloads-image][![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
```

Comment:
Now there is no build status badge? 

![status](https://cd.screwdriver.cd/pipelines/8e6fcf447ebd13f5cfbdb570e382ad8c410c2ec0/badge)